roles/download/defaults/main.yml

```diff
@@ -214,115 +221,115 @@ crictl_binary_checksum: "{{ crictl_checksums[image_arch][crictl_version] }}"
 calico_typha_image_tag: "{{ calico_typha_version }}"
-pod_infra_image_repo: "gcr.io/google_containers/pause-{{ image_arch }}"
+pod_infra_image_repo: "{{ gcr_image_repo }}/google_containers/pause-{{ image_arch }}"
 pod_infra_image_tag: "{{ pod_infra_version }}"
-install_socat_image_repo: "docker.io/xueshanf/install-socat"
+install_socat_image_repo: "{{ docker_image_repo }}/xueshanf/install-socat"
 install_socat_image_tag: "latest"
 netcheck_version: "v1.0"
-netcheck_agent_image_repo: "quay.io/l23network/k8s-netchecker-agent"
+netcheck_agent_image_repo: "{{ quay_image_repo }}/l23network/k8s-netchecker-agent"
 netcheck_agent_image_tag: "{{ netcheck_version }}"
-netcheck_server_image_repo: "quay.io/l23network/k8s-netchecker-server"
+netcheck_server_image_repo: "{{ quay_image_repo }}/l23network/k8s-netchecker-server"
 netcheck_server_image_tag: "{{ netcheck_version }}"
-weave_kube_image_repo: "docker.io/weaveworks/weave-kube"
+weave_kube_image_repo: "{{ docker_image_repo }}/weaveworks/weave-kube"
 weave_kube_image_tag: "{{ weave_version }}"
-weave_npc_image_repo: "docker.io/weaveworks/weave-npc"
+weave_npc_image_repo: "{{ docker_image_repo }}/weaveworks/weave-npc"
 weave_npc_image_tag: "{{ weave_version }}"
-contiv_image_repo: "docker.io/contiv/netplugin"
+contiv_image_repo: "{{ docker_image_repo }}/contiv/netplugin"
 contiv_image_tag: "{{ contiv_version }}"
-contiv_init_image_repo: "docker.io/contiv/netplugin-init"
+contiv_init_image_repo: "{{ docker_image_repo }}/contiv/netplugin-init"
 contiv_init_image_tag: "latest"
-contiv_auth_proxy_image_repo: "docker.io/contiv/auth_proxy"
+contiv_auth_proxy_image_repo: "{{ docker_image_repo }}/contiv/auth_proxy"
 contiv_auth_proxy_image_tag: "{{ contiv_version }}"
-contiv_etcd_init_image_repo: "docker.io/ferest/etcd-initer"
+contiv_etcd_init_image_repo: "{{ docker_image_repo }}/ferest/etcd-initer"
 contiv_etcd_init_image_tag: latest
-contiv_ovs_image_repo: "docker.io/contiv/ovs"
+contiv_ovs_image_repo: "{{ docker_image_repo }}/contiv/ovs"
 contiv_ovs_image_tag: "latest"
-cilium_image_repo: "docker.io/cilium/cilium"
+cilium_image_repo: "{{ docker_image_repo }}/cilium/cilium"
 cilium_image_tag: "{{ cilium_version }}"
-cilium_init_image_repo: "docker.io/cilium/cilium-init"
+cilium_init_image_repo: "{{ docker_image_repo }}/cilium/cilium-init"
 cilium_init_image_tag: "2019-04-05"
-cilium_operator_image_repo: "docker.io/cilium/operator"
+cilium_operator_image_repo: "{{ docker_image_repo }}/cilium/operator"
 cilium_operator_image_tag: "{{ cilium_version }}"
-kube_ovn_db_image_repo: "docker.io/kubeovn/kube-ovn-db"
-kube_ovn_node_image_repo: "docker.io/kubeovn/kube-ovn-node"
-kube_ovn_cni_image_repo: "docker.io/kubeovn/kube-ovn-cni"
+kube_ovn_db_image_repo: "{{ docker_image_repo }}/kubeovn/kube-ovn-db"
+kube_ovn_node_image_repo: "{{ docker_image_repo }}/kubeovn/kube-ovn-node"
+kube_ovn_cni_image_repo: "{{ docker_image_repo }}/kubeovn/kube-ovn-cni"
 kube_ovn_controller_image_repo: "kubeovn/kube-ovn-controller"
```

Comment:
`{{ docker_image_repo }}/kubeovn/kube-ovn-controller `  
I think this needs to be fixed.

Reply:
Good catch!

Reply:
Fixed in the latest commit